### PR TITLE
Develop mm docs various prettier

### PR DESCRIPTION
**Pull request by mmattar**
### Proposed change(s)

Running prettier formatting as a follow-up PR to #3775

Use "Display the rich diff" for convenience. 

There should be only one real change:
- Renamed section “Custom Metrics from C#” to “Custom Metrics from Unity”

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [X] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [X] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: com.unity.ml-agents/CHANGELOG.md [hunk 1]
```diff
@@ -1,148 +1,245 @@
...
+  `SideChannelUtils.GetSideChannel<FloatPropertiesChannel>()` instead.
+- Removed the multi-agent gym option from the gym wrapper. For multi-agent
+  scenarios, use the [Low Level Python API](../docs/Python-API.md).
+- The low level Python API has changed. You can look at the document
+  [Low Level Python API documentation](../docs/Python-API.md) for more
+  information. If you use `mlagents-learn` for training, this should be a
+  transparent change.
+- Added ability to start training (initialize model weights) from a previous run
+  ID. (#3710)
+- The internal event `Academy.AgentSetStatus` was renamed to
```
Comment (anupambhatnagar) [inline com.unity.ml-agents/CHANGELOG.md:28]: each changelog line should have the corresponding PR# that was responsible for the change. this also applies to various lines below.

File: docs/Using-Virtual-Environment.md [hunk 2]
```diff
@@ -1,39 +1,52 @@
 # Using Virtual Environment
 
 ## What is a Virtual Environment?
-A Virtual Environment is a self contained directory tree that contains a Python installation
-for a particular version of Python, plus a number of additional packages. To learn more about
-Virtual Environments see [here](https://docs.python.org/3/library/venv.html).
+
+A Virtual Environment is a self contained directory tree that contains a Python
```
Comment (anupambhatnagar) [inline docs/Using-Virtual-Environment.md:5]: @mmattar Are you setting a new default line width for markdown files? If yes, how is it ensured that all files follow the same convention?

Comment (anupambhatnagar) [inline docs/Using-Virtual-Environment.md:5]: @mmattar are you setting a new line width for md files? If yes, how do we ensure that the same convention is followed in all md files?

File: com.unity.ml-agents/CHANGELOG.md (hunk 1 above)
Comment (mmattar) [reply com.unity.ml-agents/CHANGELOG.md:28]: Good call out. This is just a formatting PR - we should do a separate PR where we update those.

File: docs/Using-Virtual-Environment.md (hunk 2 above)
Comment (mmattar) [reply docs/Using-Virtual-Environment.md:5]: This is a one-pass clean-up. Afterward, we'll evaluate if we use Prettier or not. We have a pre-commit config that we'll enable.

Comment (anupambhatnagar) [reply docs/Using-Virtual-Environment.md:5]: If we decide not to use prettier in the future, how do you plan to undo changes which have been incorporated by prettier in this PR?

File: com.unity.ml-agents/CHANGELOG.md (hunk 1 above)
Comment (anupambhatnagar) [reply com.unity.ml-agents/CHANGELOG.md:28]: Could you please create a branch with this particular file and tag relevant people to make the required changes? We should also mark it as a requirement for 1.0 release.

File: docs/Using-Virtual-Environment.md (hunk 2 above)
Comment (mmattar) [reply docs/Using-Virtual-Environment.md:5]: We will either use prettier, another tool or continue doing a best-effort. What we get out of using Prettier now is defining the standard and rolling it out.

File: com.unity.ml-agents/CHANGELOG.md (hunk 1 above)
Comment (mmattar) [reply com.unity.ml-agents/CHANGELOG.md:28]: 👍 pinging @chriselion

Comment (chriselion) [reply com.unity.ml-agents/CHANGELOG.md:28]: Logged https://jira.unity3d.com/browse/MLA-921. Will handle it later.

File: docs/Using-Virtual-Environment.md (hunk 2 above)
Comment (anupambhatnagar) [reply docs/Using-Virtual-Environment.md:5]: sounds good. Are we apply prettier formatting to all markdown files instead of just a handful for now?

Comment (mmattar) [reply docs/Using-Virtual-Environment.md:5]: One PR at a time. I'll essentially be touching most of the files due to the restructuring.

**Pull request merged**